nodeadm/internal/system/networking.go

```diff
@@ -89,11 +96,79 @@ func (a *networkingAspect) ensureEKSNetworkConfiguration(cfg *api.NodeConfig) er
 	return nil
 }
 
+func (a *networkingAspect) ensureMulticardNetworkConfiguration(cfg *api.NodeConfig) error {
+	var networkRestartRequired bool
+	routeTableId := 1001
```

Comment:
how is this routeTableId chosen to be starts with 1001 for those multi-card ENIs? is it chosen to align with AL2?

Reply:
I chose it to align with AL2.

Reply:
i'm ok with the 1000 route table thing, but prefer to align with AL2023's default behavior whenever possible. e.g. what's the route table routes for those enis set to if launch on a normal AL2023 instead of EKS's ones. i have no idea why 1000 were choosen for EKS Al2.

Reply:
instead of have a hacky "1000" route table, unless there are reasons. the major concern i have is this hard-coded 1000 might conflicts with other products who uses secondary route tables(vpc-cni for example). So the closer it aligns with AL2023's default behavior, the less surprise to customers

Reply:
As we discussed offline, the reason for having the routes in separate route tables was because I was aligning with what we have in AL2. However nowadays CNI actually [skips](https://github.com/aws/amazon-vpc-cni-k8s/pull/1396/files#diff-29a659a6a9cbfa72a4a83016bef85bdf10a07402f34f7d73fa7fa5eb275a8ec1) non-zero cards so we'll go with the approach of adding the routes for non-zero cards in the main routing table.